### ISSUE TITLE
HEAD won't render

### ISSUE THREAD

**Issue by jonjab** [UCSBCarpentry/2019-10-24-gitbash#1]
EventBrite and other custom HEAD info on index.md is NOT rendering on the website.  WHY?!?

**Comment by ilessing** [comment]
I cloned the repo to my local machine and then I ran `jekyll serve`  (Note: I already had Jekyll installed on my mac)
Jekyll threw the following error:
```
YAML Exception reading 
  2019-10-24-gitbash/index.md: 
(<unknown>): did not find expected ',' or ']' 
while parsing a flow sequence at line 13 column 9 
```
So... it looks like a missing double quote in `index.md` on line 13.

**Comment by ilessing** [comment]
@kristi-sara fixed this with 9df90fe784fb0
